src/includes/class_reservationPermissions.php

```diff
@@ -413,10 +467,16 @@ public function insertCSVFile() {
     }
   }
 
+  // multiDelete
+  // ==================================================
+  // Name : Tracy McCormick
+  //
+  // Description:
+  // Function performs a delete on all records passed from the form
   public function multiDelete($items = null){
     try{
-  		foreach ($items as $reservationID){
-  			self::deleteRecord($reservationID);
+  		foreach ($items as $id){
+  			$this->deleteRecord($id);
```

Comment:
Why in some places in these commits ar eyou expanding the $id variable to be more descriptive and then in others making them less descriptive? 

Err on the side of more descriptive. What type of ids are these?